conf/machine/colibri-imx7-nand.conf

```diff
@@ -1,7 +1,7 @@
 #@TYPE: Machine
-#@NAME: Toradex Colibri iMX7D/S
+#@NAME: Toradex Colibri iMX7D/S (NAND)
 #@SOC: i.MX 7Dual / i.MX 7Solo
-#@DESCRIPTION: Machine configuration for Toradex Colibri iMX7 SOM
+#@DESCRIPTION: Machine configuration for Toradex Colibri iMX7 SOM (NAND)
 #@MAINTAINER: Max Krummenacher <max.krummenacher@toradex.com>
 
 MACHINEOVERRIDES =. "mx7:mx7d:"
```

Comment:
@fbertux we need to add the old machine name as an override so we don't break existing code.